config/leviton/vrcs2.xml

```diff
@@ -1,21 +1,45 @@
+    <MetaDataItem name="OzwInfoPage">http://www.openzwave.com/device-database/001D:0243:1102</MetaDataItem>
     <MetaDataItem name="ProductPic">images/leviton/vrcs2.png</MetaDataItem>
-    <MetaDataItem id="0206" name="ZWProductPage" type="0201">https://products.z-wavealliance.org/products/288/</MetaDataItem>
-    <MetaDataItem id="0206" name="FrequencyName" type="0201">U.S. / Canada / Mexico</MetaDataItem>
-    <MetaDataItem name="Description">The Vizia RF + 4-Button Zone Controller with Switch (VRCZ4-MR) and 4-Button Scene Controller with Switch (VRCS4-MR) are perfect for retrofit applications because they replace an existing switch giving the user control of the local load as well as the benefit of scene or zone control.
- Item Description
-Vizia RF + 4-Button Zone Controller with Switch for Multi-Location Control with IR Remote Capability. White face assembled on device, ivory and light almond faces included.</MetaDataItem>
-    <MetaDataItem name="ProductPage">http://www.leviton.com/OA_HTML/ProductDetail.jsp?partnumber=VRCZ4-MRZ&amp;section=44147&amp;minisite=10251</MetaDataItem>
-    <MetaDataItem id="0206" name="Identifier" type="0201">VRCZ4-MR</MetaDataItem>
-    <MetaDataItem name="ProductSupport">http://www.leviton.com/OA_HTML/Home.jsp?minisite=10251&amp;respid=22372</MetaDataItem>
-    <MetaDataItem name="Name">VRCZ4-MR</MetaDataItem>
+    <MetaDataItem id="0243" name="ZWProductPage" type="1102">https://products.z-wavealliance.org/products/286/</MetaDataItem>
+    <MetaDataItem id="0243" name="FrequencyName" type="1102">U.S. / Canada / Mexico</MetaDataItem>
+    <MetaDataItem id="0243" name="Identifier" type="1102">VRCS2-MRZ</MetaDataItem>
     <ChangeLog>
       <Entry author="Justin Hammond - Justin@dynam.ac" date="03 May 2019" revision="1">Initial Metadata Import from Z-Wave Alliance Database - https://products.z-wavealliance.org/products/286/xml</Entry>
       <Entry author="Justin Hammond - Justin@dynam.ac" date="03 May 2019" revision="2">Updated Metadata Import from Z-Wave Alliance Database - https://products.z-wavealliance.org/products/287/xml</Entry>
       <Entry author="Justin Hammond - Justin@dynam.ac" date="03 May 2019" revision="3">Updated Metadata Import from Z-Wave Alliance Database - https://products.z-wavealliance.org/products/288/xml</Entry>
+      <Entry author="Jeff Brown - jeff.brown@gmail.com" date="29 June 2020" revision="4">Fixed metadata, added groups and indicators</Entry>
     </ChangeLog>
   </MetaData>
-  <CommandClass id="112"/>
+
+  <!-- Association Groups -->
+  <CommandClass id="133">
+    <Associations num_groups="2">
+      <!-- Each button toggles an individual scene on or off.  Multiple scenes can be active at once.
+           By default the device will associate groups 1 and 2 with the local loads.
+           Do not auto associate the controller with group 1. -->
+      <Group index="1" max_associations="232" label="Button 1" auto="false"/>
```

Comment:
If you do not auto associate, how does the controller get updates about state change? Maybe all Groups should be auto=true (group 1 is implicit)

Reply:
The scene controller and the loads are represented as different zwave nodes using multichannel encapsulation.

The scene controller has factory default associations with the local loads (relays) on the device.  There's no need to associate the primary controller unless you want the primary controller to respond to scene changes itself.  In my case, I just want the VRCS2 loads to be locally controlled (so they're very responsive).

The loads appear as separate binary switches.  They seem to get auto-associated (somehow?) with the primary controller so they do report their state changes as expected.

Aside: My old Vera automation system had a nasty habit of purging factory default associations when setting up a new device.  In the case of the VRCS2, it would remove the associations for the local loads and effectively break the switch behavior unless manually reconfigured.  It took ages to figure out what was happening there.